src/runtime.jl

```diff
@@ -99,7 +94,22 @@ function init(runtime_env::Union{RuntimeEnv,Nothing}=nothing;
     job_config = JobConfig(RuntimeEnvInfo(runtime_env), metadata)
     serialized_job_config = _serialize(job_config)
 
-    ray_jll.initialize_driver(args..., job_id, logs_dir, serialized_job_config)
+    raylet, store, node_port = get_node_to_connect_for_driver()
+
+    @info begin
```

Comment:
Lost this comment in the shuffle
```suggestion
    # TODO: downgrade to debug
    # https://github.com/beacon-biosignals/Ray.jl/issues/53
    @info begin
```